test/lib_i2s/i2s_slave_test/i2s_slave_test.cmake

```diff
@@ -16,6 +16,10 @@ set(APP_LINK_OPTIONS
     -target=XCORE-AI-EXPLORER
 )
 
+# Compile main.c which contains the i2s_callback_group_t functions in O3 mode. Needed for passing the
+# test_i2s_basic_slave[4ch_in,4ch_out-32b] and test_i2s_basic_slave[4ch_in,4ch_out-16b] tests
+set_source_files_properties(${CMAKE_CURRENT_LIST_DIR}/src/main.c  PROPERTIES COMPILE_FLAGS "-O3")
```

Comment:
This test is intended to check that the component is capable of operating using these settings, which it does - which is good. I would caution that this does highlight that the backpressure tolerance for the component in these settings is lower than we expected it to be - it would be useful to be able to profile these in the backpressure "test".